src/App.tsx

```diff
@@ -15,71 +15,83 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface State {
+  selectedGood: string | '';
```

Comment:
Empty string is also a string
```suggestion
  selectedGood: string;
```

Reply:
change this one